console/src/components/Common/WarningSymbol/WarningSymbol.tsx

```diff
@@ -4,10 +4,16 @@ import OverlayTrigger from 'react-bootstrap/lib/OverlayTrigger';
 
 import styles from './WarningSymbol.scss';
 
-const WarningSymbol = ({
+export interface WarningSymbolProps {
+  tooltipText: string;
+  tooltipPlacement?: string;
```

Comment:
We can narrow it to `'left' | 'right' | 'top' | 'bottom'`.

Reply:
Fixed. Thanks for the review 🙏